crates/miden-lib/asm/agglayer/account_components/bridge_in.masm

```diff
@@ -0,0 +1,17 @@
+use.agglayer::global_exit_tree
+
+#! Checks the validity of the GET proof
+#!
+#! Inputs:  []
+#! Outputs: [is_valid_claim_proof]
+#!
+#! Invocation: exec
+pub proc check_claim_proof
```

Comment:
Similar to my other comments: let's define what inputs to this procedure should be. Off the top of my head, these are:

- The bridge message hash: 8 elements (or we can compress to 4 if needed).
- GER index (so that we know how to find the right root to validate against).
- `is_l1_claim` - a flag indicating whether the bridge message is from L1 or another L2.
- Leaf info - the indexes of the leaf either in the mainnet tree or local exit trees of another L2.
- Merkle path commitment (4 elements).